### ISSUE TITLE
Compiled Archives

### ISSUE THREAD

**Issue by dselman** [accordproject/cicero#328]
To support efficient execution of Cicero archives it is useful to be able to pre-compile them to a target runtime offline/once.

Also see: #258 

This metadata field can then be used by the `Template.fromArchive` and `Template.fromDirectory` code to either reject loading the archive, or to specialise the handling of the resources within the archive.

The proposed format is:

```
"accordproject": {
        "template": "contract",
        "ergo": "0.7.3",
        "cicero": "^0.11.1"
        "runtime": "cicero"
    },
```

**Comment by jeromesimeon** [comment]
Great idea. Note that we could consider removing the `language` tag in the `package.json` after that, since it would always be Ergo.

**Comment by jeromesimeon** [comment]
Implemented in 0.12
